### DISCUSSION TITLE
TrustWallet on Android issue

### DISCUSSION THREAD

**Discussion by pmo-van-018** [trustwallet/wallet-core#3533] [Q&A]
**Describe the bug**
TrustWallet on Android always add the blob: prefix into my script src, so It make the url become invalid. E.g: my url was https://challenge.cloudflare.com/ => blob:https://challenge/....

**Expected behavior**
It shoud be call my url without any issues

**Screenshots**
![telegram-cloud-photo-size-5-6192915371783207219-y](https://github.com/trustwallet/wallet-core/assets/121281935/a3a29d3d-be48-4cd8-9307-04f545b342d6)

**Additional context**
Version app: 8.1.1

@hewigovens Please help to check, thanks

**Comment by pmo-van-018** [comment]
Anyone help me? Its make my app crash now

**Comment by vcoolish** [comment]
> Anyone help me? Its make my app crash now

https://challenge.cloudflare.com/ what do you expect to see here? it doesn't load for me anywhere

**Comment by pmo-van-018** [comment]
> > Anyone help me? Its make my app crash now
> 
> https://challenge.cloudflare.com/ what do you expect to see here? it doesn't load for me anywhere

It just an example link @vcoolish , I can not show full link ^^

**Comment by vcoolish** [comment]
> > > Anyone help me? Its make my app crash now
> 
> > 
> 
> > https://challenge.cloudflare.com/ what do you expect to see here? it doesn't load for me anywhere
> 
> 
> 
> It just an example link @vcoolish , I can not show full link ^^

Cannot help you without full link. My assumption is that you website does load something from this cloudflare link which just fails with dns probe. Doesn't really matter that it displays blob prefix its ui stuff
